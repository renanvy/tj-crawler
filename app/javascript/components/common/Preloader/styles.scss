.Preloader {
  display: flex;
  justify-content: center;
}