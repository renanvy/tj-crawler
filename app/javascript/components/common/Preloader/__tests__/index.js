import React from "react";
import renderer from "react-test-renderer";
import Preloader from "../index";

describe("Preloader", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Preloader />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
