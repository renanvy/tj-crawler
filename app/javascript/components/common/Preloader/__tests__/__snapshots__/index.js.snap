// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader snapshot renders 1`] = `
<div
  className="Preloader"
>
  <div
    className="preloader-wrapper big active"
  >
    <div
      className="spinner-layer spinner-blue-only"
    >
      <div
        className="circle-clipper left"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="gap-patch"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="circle-clipper right"
      >
        <div
          className="circle"
        />
      </div>
    </div>
  </div>
</div>
`;
